src/files/file-preview/FilePreview.js

```diff
@@ -51,8 +51,9 @@ class FilesPreview extends React.Component {
         return <img className={className} alt={stats.name} src={src} />
       default:
         const cantPreview = (
-          <div className='b'>
-            Sorry, this file can't be previewed <span role='img' aria-label='sad'>😢</span>
+          <div>
+            <p className='b'>Sorry, this file can't be previewed <span role='img' aria-label='sad'>😢</span></p>
+            <p>Try <a href={src} download target='_blank' className='underline-hover navy-muted' >downloading</a> it instead.</p>
```

Comment:
Note: `download` won't work for the most of time because `{src}` points at a different Origin, so I've added `target='_blank'` as an universal fallback.